### PR TITLE
Allow to call `timm` models with `timm/model_name` on `BackboneModelRegistry`

### LINKED ISSUE #612: Investigate a more user friendly `timm` model API

### PR DESCRIPTION
**Pull request by ioangatop**
Closes #612

### CODE REVIEW

File: src/eva/vision/models/networks/backbones/universal/timm.py
Comment (nkaenzig) [inline src/eva/vision/models/networks/backbones/universal/timm.py:1]: Please update also the documentation in:
https://kaiko-ai.github.io/eva/latest/user-guide/advanced/custom_checkpoints/

**Review by nkaenzig — COMMENTED** [review]
Nice idea!

File: src/eva/vision/models/networks/backbones/universal/timm.py
[the remark this answers is not in the document]
Comment (ioangatop) [reply src/eva/vision/models/networks/backbones/universal/timm.py:1]: nice good one; done!

Comment (ioangatop) [reply src/eva/vision/models/networks/backbones/universal/timm.py:1]: done 🙏

**Review by nkaenzig — APPROVED** [review]
LGTM

**Pull request merged**